app/views/support/schools/opt_outs/edit.html.erb

```diff
@@ -0,0 +1,26 @@
+<%- title = t('page_titles.support.schools.opt_outs.edit') %>
+<%- content_for :title, title %>
+<%- content_for :before_content do %>
+  <%= govuk_link_to('Back', support_school_path(@school), class: 'govuk-back-link') %>
+<%- end %>
+
+<div class="govuk-grid-row">
```

Comment:
I think we need something on this page about this being explicitly about not wanting allocations as the reason for opting out.

Reply:
I've added the following above the form and the screenshot has been updated
```
By opting outing of communications a user from the school has expressed that they no longer wish to use the remainder of their allocation. This will prevent us from sending any further updates to them about unclaimed devices.
```